src/features/device/pca10153.json

```
@@ -80,6 +80,38 @@
                     }
                 ]
             },
+            {
+                "name": "nRF Cloud Multi-Service",
```

```suggestion
                "name": "nRF Cloud multi-service",
```
Consistency with the sample naming.